services/motion/builtin/move_request.go

```diff
@@ -538,6 +562,16 @@ func (ms *builtIn) newMoveOnMapRequest(
 	}
 	limits = append(limits, referenceframe.Limit{Min: -2 * math.Pi, Max: 2 * math.Pi})
 
+	// naive check that the destination is within slam's limits
```

Comment:
Why are we adding this? Is this fixing a bug?

Reply:
The intended functionality of this code segment is to make sure that the selected point we navigate to is within the bounds of the already existing slam map we are using.
- In the scenario where the robot is in an octagon, I believe it makes sense to impose limits as to where the robot may move.
- However, this functionality wouldn't play well, I think, if we are in explore mode. In this scenario, I imagine, we would be building a slam map and potentially navigating to a point which does not yet exist on the slam map. This code segment would prevent us from doing that.
-------
Here, I am not sure sure what is expected .. Potentially we should check in with product on this?
@biotinker @raybjork what're your thoughts on this?

Reply:
Leaving around for @biotinker & @raybjork to answer

Reply:
If its out of bounds motionplanning should be returning an error.  So I think this is redundant work

Reply:
will remove.